plugins/uefi/uefi.conf.in

```diff
@@ -2,4 +2,4 @@
 
 # For fwupdate 10+ allow overriding
 # the compiled EFI system partition path
-#OverrideESPMountPoint=
+#OverrideESPMountPoint=@bootdir@
```

Comment:
Do we want to uncomment it too?

Reply:
Funny, your comment makes me feel like it should be uncommented now too.  Uncommented means that we don't care what fwupdate did.  If someone compiled it funny or not.  Since it comes through fwupd, fwupd always configures it.

I guess lets flip a coin :)